web/components/Home/Section.js

```diff
@@ -7,7 +7,7 @@ export default function Section({ children }) {
   return (
     <div className="bg-gradient-to-br from-blue-900 to-blue-500 relative w-full pb-20 overflow-x-hidden text-white">
       {children}
-      <div className="absolute bottom-0 flex">
+      <div className="-bottom-1 absolute flex">
```

Comment:
I am not sure stating the class name with "-" is the better way, and at the same time I know it's represented a size with a minus, I've tried to search about it but couldn't found the best way to define sizes with minus values in tailwind

Reply:
that's not a class I made, that's how tailwind does negative values